src/index.ts

```diff
@@ -0,0 +1,4 @@
+export type { VueDriftWidgetConfig, DriftWidget } from './types'
+export { createDrift } from './install'
+export { loadScript } from './utils'
+export { useDrift } from './usedrift'
```

Comment:
maybe rename to `useDriftAPI` or `useDriftWidget` ?

Reply:
changed in f55c995